all.json

```diff
@@ -11399,6 +11399,49 @@
 		"zinotiger.com",
 		"zipwallet.net",
 		"zonefix.net",
-		"zwalletconnect.com"
+		"zwalletconnect.com",
+		"walletsbridges.com",
+		"wcaservice.com",
+		"walletprotectauth.com",
+		"wallsynchvalidatevd.com",
+		"smartsyncnode.com",
+		"walletsconnects.cloud",
+		"vvalletconnect.com",
+		"elsalvadorairdrop.com",
+		"weyu.pro",
+		"wallectsyncfixs.com",
+		"raymasterbatch.com",
+		"duckdns.org",
+		"authenticatedappwallets.com",
+		"mydevsite.biz",
+		"app-sync.net",
+		"versalunigen.buzz",
+		"refreshwallet.company",
+		"openconnect.company",
+		"mmxcs.com",
+		"univingen.click",
+		"unitsgen.buzz",
+		"unitsgeu.buzz",
+		"walletdappconnect.app",
+		"scannoderestore.com",
+		"cronos-active.com",
+		"apollovelo.sbs",
+		"xxxcinnexxt.online",
+		"unitigent.buzz",
+		"walletconnect-sale.com",
+		"vaultftconnectserver.com",
+		"uniqr-c.com",
+		"playmedia.ma",
```

Comment:
This one seems above-board - were they sub-domain hijacked?

Reply:
Going to take that off for now.